docs/dev/development.rst

```diff
@@ -63,23 +65,27 @@ To proceed, stage the new modifications and proceed with your Git commit.
 Running tests
 =============
 
-One way to test the library is by running pytest_ from the root of the source repository:
+To test the library, run tox_, which tests the library the same way that the CI workflow does:
 
 .. code-block:: sh
 
-   pytest
+   tox
 
-You can also run tox_, which tests the library the same way that the CI workflow does:
+To see a listing of test environments, run:
 
 .. code-block:: sh
 
-   tox
+   tox -av
 
-To see a listing of test environments, run:
+tox will start a PostgreSQL container, which is required for some tests.
+You can also run pytest_ from the root of the source repository:
 
 .. code-block:: sh
 
-   tox -av
+   pytest
```

Comment:
At this point I'm generally never running pytest directly; you can always pass args to pytest, like a specific module with `--`, like

```
tox -e docker -- tests/some/module_test.py
```

Reply:
Good call, I've deleted this bit entirely and instead added an example of running a specific test with `tox`.